sdk/entra/WebJobs.Extensions.AuthenticationEvents/src/WebJobs.Extensions.AuthenticationEvents/Framework/AuthEventMetadata.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Net.Http;
+using System.Reflection;
+
+namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Framework
+{
+    /// <summary>Represents event meta-data.</summary>
+    internal class AuthEventMetadata
+    {
+        /// <summary>Gets or sets the type of the request.</summary>
+        /// <value>The type of the request.</value>
+        internal Type RequestType { get; set; }
+
+        /// <summary>Gets or sets the identifier.</summary>
+        /// <value>The identifier.</value>
+        internal EventDefinition Id { get; set; }
+
+        /// <summary>Gets or sets the request schema.</summary>
+        /// <value>The request schema.</value>
+        internal string RequestSchema { get; set; }
+
+        /// <summary>Gets or sets the response schema content.</summary>
+        /// <value>The response schema content.</value>
+        internal string ResponseSchema { get; set; }
+
+        /// <summary>Gets or sets the response template content.</summary>
+        /// <value>The response template content.</value>
+        internal string ResponseTemplate { get; set; }
+
+        internal string OpenApiDocument { get; set; }
```

Comment:
Odd that all but this one have comments...

Reply:
Done